docs-src/go/how-to-schedule-a-workflow-in-go.md

```diff
@@ -0,0 +1,233 @@
+---
+id: how-to-schedule-a-workflow-in-go
+title: How to Schedule a Workflow in Go 
+sidebar_label: Schedule
+description: Set a Schedule in Go for scheduling Workflows to run at specific times.
+tags:
+  - go
+  - developer-guide
+  - how-to
+---
+
+Scheduling [Workflows](/concepts/what-is-a-workflow) is crucial to automation.
+By scheduling [Workflow Executions](/concepts/what-is-a-workflow-execution), you can reduce manual intervention and ensure timely execution of business processes.
+
+Temporal allows you to execute multiple Schedule operations, either through code or with the [CLI tool](/concepts/what-is-the-temporal-cli).
+Read on to find out how to create and edit Schedules.
+
+:::note
+Be sure to [enable Schedules in your environment](/concepts/what-is-a-schedule#limitations) before creating any Schedules.
+:::
+
+## Create
+
+Schedules are created with the `create` action.
+For each new Schedule, tbe Temporal Server generates a unique Schedule ID.
```

Comment:
This paragraph says the server generates the schedule ID, that paragraph below says the user must pass an ID. These don't seem consistent

Reply:
Changed per @cretz correction